src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,22 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+public interface CarDao {
+
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
get on Dao should always return optional - dao's job doesn't include business logic of how to handle the case where required entity is missing, we are returning optional to the Service layer, and only there we will deal with this